### PR TITLE
Connection IDs for 0-RTT packets

### PR DESCRIPTION
**Pull request by martinthomson**
For #2398 and @mikkelfj.

### CODE REVIEW

File: draft-ietf-quic-transport.md [hunk 1]
```diff
@@ -1355,6 +1355,9 @@ long enough to fulfill the minimum size for every QUIC version it supports.
 The client populates the Source Connection ID field with a value of its choosing
 and sets the SCIL field to match.
 
+The same Destination and Source Connection ID values are used for any 0-RTT
+packets the client sends.
+
```
Comment (mikkelfj) [inline draft-ietf-quic-transport.md]: This could be read as if the ID is the same in both directions.

**Comment by mikkelfj** [comment]
Thanks

File: draft-ietf-quic-transport.md (hunk 1 above)
Comment (ianswett) [reply draft-ietf-quic-transport.md]: Agreed, this is not completely clear.  How about: "The first flight of 0-RTT packets use the same Destination and Source Connection ID values as the client's first Initial."?

Also, on 1356, "sets the SCIL field to match." is not very clear.  I'd either drop the clause or say "and sets the SCIL field to indicate the length."

**Comment by mikkelfj** [comment]
Note that section 17.2.3. 0-RTT does not mention 0-RTT CID but the following section 7.2.4. Handshake Packet does mention Handshake CID's.

**Pull request merged**